### PR TITLE
Product Upload page done with out the pricing scheme

### PR DESCRIPTION
**Pull request by sakibesrat71**

### CODE REVIEW

File: apps/frontend/src/app/app.tsx [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+import { ProductUpload } from './product-upload/ProductUpload';
+import { ProductShow } from './product-show/ProductShow';
 import { Home } from './home/home';
 import { SignIn } from './signin/sign-in';
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/signin" element={<SignIn />} />
+		<Route path="/" element={<ProductUpload />} />
```
Comment (Noman5237) [inline apps/frontend/src/app/app.tsx:12]: Fix routes

**Review by Noman5237 — CHANGES_REQUESTED** [review]
Job Well Done

File: apps/frontend/src/app/app.tsx (hunk 1 above)
Comment (Noman5237) [reply apps/frontend/src/app/app.tsx:12]: image

Comment (sakibesrat71) [reply apps/frontend/src/app/app.tsx]: Code fixed as per request

**Review by Noman5237 — APPROVED** [review]
Good work on product upload frontend

**Pull request merged**